build.fsx

```diff
@@ -476,7 +476,10 @@ Target.create "CheckFormat" (fun _ ->
     elif result.ExitCode = 99 then
         failwith "Some files need formatting, run `dotnet fake build -t Format` to format them"
     else
-        Trace.logf "Errors while formatting: %A" result.Errors)
+        Trace.logf "Errors while formatting: %A" result.Errors
+        failwith "Unknown errors while formatting")
+
+Target.create "EnsureRepoConfig" (fun _ -> Fake.Tools.Git.CommandHelper.gitCommand "" "config core.hooksPath .githooks")
```

Comment:
We should add this to the Contribution Guidelines, section https://github.com/fsprojects/fantomas/blob/master/CONTRIBUTING.md#getting-started-with-fantomas.

Reply:
Added, as well as documented in the build.fsx the purpose of this particular git call.